ray-operator/config/samples/ray-data-image-resize/ray_data_image_resize.py

```diff
@@ -0,0 +1,31 @@
+from typing import Dict
+
+import numpy as np
+import ray
+from torchvision import transforms
+
+DATA_URI = "gs://anonymous@ray-images/images"
+
+def main():
+    """
+    This is a CPU-only job that reads images from
+    Google Cloud Storage and resizes them.
+    """
+    ray.init()
+    dataset = ray.data.read_images(DATA_URI, include_paths=True)
+    dataset = dataset.map(transform_frame)
+    dataset = dataset.select_columns(["path"])
+    dataset_iter = dataset.iter_batches(batch_size=None)
```

Comment:
Will this distribute across multiple workers?

Reply:
In my understanding, Ray Data will create varying numbers of tasks or actors based on the Ray cluster resources. cc @raulchen to confirm.

You can see in my screenshot that the workload used 10/10 CPUs while it was running, so I think it is running across multiple Ray Pods.

Reply:
thanks, makes sense to me!